js/paragraphs-features.delete-confirmation.js

```diff
@@ -52,6 +52,8 @@
   Drupal.paragraphs_features.deleteConfirmHandler = function () {
     return function (event) {
       var $wrapper = $(event.target).parents('div[id*="-item-wrapper"]').first();
+      // Hiding of elements can lead to incorrect behaviour for some components.
+      Drupal.detachBehaviors($wrapper.children(), drupalSettings, 'paragraphs-features-delete-confirmation');
```

Comment:
I think it is better to first wrap the visually-hidden div before detaching, to prevent visual clutter, that could be caused by detaching behaviors.